cli/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.1 - 2023-MM-DD
```

Comment:
Tbh if we have 1.1 very soon, maybe I'd just do it there?

Reply:
Let's just change when we know the date

Reply:
Yeah but I mean, do we want to target the 1.1 branch then ?